test/helpers/running.js

```diff
@@ -60,6 +60,11 @@ async function runTest(test) {
     resetForNextTest()
     await test.setup()
     
+    // Ensure any `place_changed` events that Google will fire in response to
+    // our test setup have already passed, so that we will get accurate results
+    // regarding the final location value returned.
+    await waitAMoment(1000)
```

Comment:
Thanks for the comment, that's helpful in knowing why here...I wonder why those events are getting fired during set up?

Reply:
A couple of the tests actually trigger location change events during their set up (e.g. "Type something, see suggestions, don't select any, hit Enter, hit Tab").

Reply:
oh ok, that makes sense now